app/controllers/sessions_controller.rb

```diff
@@ -22,4 +22,4 @@ def destroy
     redirect_to root_path, notice: "Log out"
   end
 
-end
```

Comment:
Why you do changes in this file

Reply:
I had to show messages using flash , so i used flash instead of notice

Reply:
See the files changes once, Its showing you only add one space.